.github/workflows/test.yaml

```diff
@@ -0,0 +1,42 @@
+---
+name: Test
+on:
+  push:
+    branches:
+      - main
+      - develop
+  pull_request:
+
```

Comment:
Do this mean that it will run on every pull request regardless of branch?

Reply:
Yep. In the past I've found this pretty useful in the past and we can always update if we decide that it's not helpful on this project.

Reply:
That sounds good to me.